### PR TITLE
Login

### PR DESCRIPTION
**Pull request by pkuang5**
google login button

### CODE REVIEW

File: components/Website_Layout/Navbar.js [hunk 1]
```diff
@@ -26,6 +41,17 @@ export default function navbar() {
       <Link href="/MerchandisePage">
         <a style={tabStyle}>Buy our Merchandise!</a>
       </Link>
+      {loggedIn ? <p>{username}</p>:
+      <GoogleLogin
+        render={renderProps => (
+          <button onClick={renderProps.onClick} disabled={renderProps.disabled}>Login</button>
```
Comment (gmundewadi) [inline components/Website_Layout/Navbar.js:47]: I deployed to qa and it looks like the button on line 47 is always disabled. Double check` renderProps.disabled`

File: components/Website_Layout/Navbar.js [hunk 2]
```diff
@@ -26,6 +41,17 @@ export default function navbar() {
       <Link href="/MerchandisePage">
         <a style={tabStyle}>Buy our Merchandise!</a>
       </Link>
+      {loggedIn ? <p>{username}</p>:
```
Comment (gmundewadi) [inline components/Website_Layout/Navbar.js:44]: add a profile pic if the user is logged in

**Review by gmundewadi — CHANGES_REQUESTED** [review]

File: components/Website_Layout/Navbar.js (hunk 1 above)
Comment (pkuang5) [reply components/Website_Layout/Navbar.js:47]: yea u gotta have a seperate oauth key for deployment. current one only works for localhost

Comment (gmundewadi) [reply components/Website_Layout/Navbar.js:47]: You’re merging to master, have that separate oauth key setup

File: components/Website_Layout/Navbar.js (hunk 2 above)
Comment (pkuang5) [reply components/Website_Layout/Navbar.js]: done

File: components/Website_Layout/Navbar.js (hunk 1 above)
Comment (pkuang5) [reply components/Website_Layout/Navbar.js:49]: added into heroku

**Pull request merged**